doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -767,6 +767,10 @@ DFU libraries
 Modem libraries
 ---------------
 
+* :ref:`lib_at_host`:
+
+  * Changed the default termination mode from the :kconfig:option:`CONFIG_CR_TERMINATION` Kconfig option to the :kconfig:option:`CONFIG_CR_LF_TERMINATION` kconfig option.
```

Comment:
```suggestion
  * Updated to set the default termination mode in the :kconfig:option:`CONFIG_CR_LF_TERMINATION` Kconfig option instead of the ``CONFIG_CR_TERMINATION`` kconfig option.
```

Reply:
`CONFIG_CR_TERMINATION` is still a valid option, so I'll keep the format for it.